### PR TITLE
README: Specify low priority for mouse support.

### PR DESCRIPTION
**Pull request by preetmishra**
I have added the description in a new Frequently Asked Questions section. I think it would be a good place to jot down points like this over time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -359,3 +359,9 @@ This likely means that you have installed both normal and development versions o
 To ensure you run the development version:
 * If using pipenv, call `pipenv run zulip-term` from the cloned/downloaded `zulip-terminal` directory;
 * If using pip (pip3), ensure you have activated the correct virtual environment (venv); depending on how your shell is configured, the name of the venv may appear in the command prompt. Note that not including the `-e` in the pip3 command will also cause this problem.
+
+## Frequently Asked Questions
+
+### Mouse does not support X?
+
+We think of Zulip Terminal as a keyboard-centric client. Consequently, while functionality via the mouse does work in places, mouse support is not currently a priority for the project.
```
Comment (neiljp) [inline README.md:367]: There is maybe an issue for

**Comment by preetmishra** [comment]
@neiljp Thanks for taking a look. It seems plausible to not put this at the bottom. I have moved the specification under 'Troubleshooting' for now.

We could discuss extracting an FAQs section if we end up putting a bunch of queries/questions later.

**Comment by neiljp** [comment]
@preetmishra Re FAQs, I sort of meant that the existing section sort of *is* a FAQ section - I'm not sure if we should rename it, but that looks good for now.

We could perhaps mention the 'keyboard-centric' nature as one key aspect in a fuller description, as per recent discussions on adding a fuller description for the project at the top of the README.

**Comment by neiljp** [comment]
@preetmishra Merged with slight changes as edfdb7e6b3cddd843e93448dd19ba4d6e2d74cf5 to make it clear what 'X' is, and slight clarify commit message. Thanks! :tada:

**Pull request closed without merging**